packages/customer-account-ui-extensions/src/extension-points/standard-api/navigation.ts

```diff
@@ -0,0 +1,30 @@
+export type NavigationType = 'push' | 'replace' | 'traverse';
+
+export interface NavigationOptions {
+  state?: Record<string, any>;
+  type: 'push' | 'replace';
+}
+
+export interface NavigationHistoryEntry {
+  key: string;
+  url: string;
+  state: Record<string, any>;
+}
+
+export interface NavigationCurrentEntryChangeEvent {
+  type: NavigationType;
```

Comment:
```suggestion
  navigationType: NavigationType;
```

https://developer.mozilla.org/en-US/docs/Web/API/NavigationCurrentEntryChangeEvent#:~:text=was%20navigated%20from.-,navigationType,-Read%20only